python_tests/artifact/test_backend.py

```diff
@@ -177,3 +179,63 @@ def test_successful_trial_artifact_retrieval() -> None:
         )
         assert status == 200
         assert body == b"dummy_content"
+
+
+DUMMY_DATA = f"data:text/plain; charset=utf-8,{base64.b64encode(b'dummy_content').decode('utf-8')}"
```

Comment:
Could you make the variable name a bit more specific like below?

```suggestion
DUMMY_DATA_URL = f"data:text/plain; charset=utf-8,{base64.b64encode(b'dummy_content').decode('utf-8')}"
```